modules/osd-aws-privatelink-firewall-prerequisites.adoc

```diff
@@ -1,14 +1,15 @@
 // Module included in the following assemblies:
 //
 // * rosa_getting_started/rosa-aws-prereqs.adoc
+// * rosa_planning/rosa-sts-aws-prereqs.adoc
```

Comment:
Following the ROSA restructure, the module looks to be included now in the following assemblies in this feature branch:

// Module included in the following assemblies:
//
// * rosa_install_access_delete_clusters/rosa_getting_started_iam/rosa-aws-prereqs.adoc
// * rosa_planning/rosa-sts-aws-prereqs.adoc

Can you please also update the `// * rosa_getting_started/rosa-aws-prereqs.adoc` reference to `// * rosa_install_access_delete_clusters/rosa_getting_started_iam/rosa-aws-prereqs.adoc`? Thanks!